runtime/jvmti/jvmtiHelpers.c

```diff
@@ -128,23 +152,50 @@ getVMThread(J9VMThread * currentThread, jthread thread, J9VMThread ** vmThreadPt
 	*vmThreadPtr = targetThread;
 	if (targetThread != NULL) {
 		++(targetThread->inspectorCount);
+#if JAVA_SPEC_VERSION >= 19
+		if (isVirtualThread) {
+			U_64 vthreadInspectorCount = J9OBJECT_U64_LOAD(currentThread, threadObject, vm->virtualThreadInspectorCountOffset);
+			J9OBJECT_U64_STORE(currentThread, threadObject, vm->virtualThreadInspectorCountOffset, ++vthreadInspectorCount);
```

Comment:
Do not use ++ here, just add 1.

Reply:
Remember it's always illegal to use side-effect expressions in macros.

Reply:
Updated.